src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -132,12 +132,24 @@ class FormStartControls extends React.Component {
         </div>
       );
     }
-
-    return (
+    const startText = this.props.startText || 'Get Started';
```

Comment:
Can `Get Started` be moved into `defaultProps`?

Reply:
Done